modules/oadp-deleting-backups.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-deleting-backups_{context}"]
+= Deleting backups
+
+You can remove backup files by deleting the `Backup` custom resource (CR).
+
+[WARNING]
+====
+After you delete the `Backup` CR and the associated object storage data, you cannot recover the deleted data. 
+====
+
+.Prerequisites
+
+* You created a `Backup` CR.
+* You know the name of the `Backup` CR and the associated namespace. 
+
+.Procedure
+
+* Choose one of the following actions to delete the `Backup` CR:
+
+** To delete the `Backup` CR and keep the associated object storage data, issue the following command:
++
+[source,terminal]
+----
+$ oc delete backup <backup_CR_name> -n <velero_namespace>
+----
+
```

Comment:
```suggestion
[source,terminal]
----
$ oc delete backup <backup_CR_name> \// <1>
    -n <velero_namespace> <2>
----
<1> Specifies the name of the `Backup` custom resource.
<2> Specifies the namespace that contains the `Backup` custom resource.
```
I'd recommend doing something like this to be as explicit as possible. This should result in a code block like this: 
https://docs.openshift.com/container-platform/4.13/installing/installing_azure/enabling-user-managed-encryption-azure.html#preparing-disk-encryption-sets

Reply:
I went with the definition list, because I personally think this style offers a cleaner source file. I'm not a fan of placing short commands over two lines.